doc-content/enterprise-only/openshift/secrets-kie-create-proc.adoc

```diff
@@ -3,7 +3,13 @@
 
 OpenShift uses objects called `Secrets` to hold sensitive information, such as passwords or keystores. See the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/dev-guide-secrets[Secrets chapter] in the OpenShift documentation for more information.
 
+ifeval::["{context}"!="openshift-ansible-playbook"]
 You must create an SSL certificate for {KIE_SERVER} and provide it to your OpenShift environment as a secret.
+endif::[]
+ifeval::["{context}"=="openshift-ansible-playbook"]
+{KIE_SERVER} uses an SSL certificate to provide HTTPS access. The deployment can create the secret automatically. However, in production environments an automatically created secret can cause access issues. To avoid these issues, create an SSL certificate for {KIE_SERVER} and provide it to your OpenShift environment as a secret.
```

Comment:
Automatically created secrets should not be used in production environment. It is same as for example secrets (used for templates).